packages/arch/newlib/PKGBUILD

```diff
@@ -27,7 +24,7 @@ prepare() {
 build() {
   cd "$srcdir/newlib-build"
   $srcdir/newlib/configure --target=$_target --prefix=/usr --enable-newlib-iconv
-  PATH=$srcdir/xargo/bin:$PATH XARGO_RUST_SRC="$srcdir/rust/src" make all
+  PATH=$srcdir/xargo/bin:$PATH make -j $(nproc) all
```

Comment:
I don't think `-j` should be included. The convention on Arch seems to be to set this in `MAKEFAGS` in `/etc/makepkg.conf`: https://wiki.archlinux.org/index.php/Makepkg#Parallel_compilation

Reply:
Ah, sorry, I wasn't aware of this, should I make the PR or would you like to?

Reply:
I've gone ahead and make the PR https://github.com/redox-os/libc/pull/53